resources/assets/components/utilities/ScholarshipCard/ScholarshipCard.js

```diff
@@ -13,6 +14,7 @@ import {
   trackAnalyticsEvent,
   getPageContext,
 } from '../../../helpers/analytics';
+import { isAuthenticated } from '../../../selectors/user';
```

Comment:
This is a touch confusing, but the `selectors` are specific to our Redux implementation and rely on the global `state` (and a container) to function properly. 

We can import the shinier `isAuthenticated` from the `/helpers/auth.js`file which will let us simply evaluate the `window.AUTH` object without Redux magic.
```suggestion
import { isAuthenticated } from '../../../helpers/auth';
```